common/bytes.go

```diff
@@ -160,3 +160,8 @@ func Int64ToByteBigEndian(number uint64) []byte {
 
 	return enc
 }
+
+// Int64ToByteBigEndian encodes a number as big endian uint64
```

Comment:
```suggestion
// ByteBigEndianToInt64 encodes a number as big endian uint64
```

Reply:
As you know, some converting function notates `uint64` as `int64`. How do you think about changing them to more precise names?

Reply:
I changed it :) I also renamed :
* `Int64ToByteLittleEndian` -> `Uint64ToByteLittleEndian`
* `Renamed Int64ToByteBigEndian` -> Uint64ToByteBigEndian`